tqftpserv.c

```diff
@@ -81,7 +81,7 @@ static ssize_t tftp_send_data(struct tftp_client *client,
 	if (response_size != 0) {
 		/* Header (4 bytes) + data size */
 		send_len = 4 + response_size;
-		if (send_len > p - buf) {
+		if (send_len > (unsigned long long)(p - buf)) {
```

Comment:
This is called ptrdiff_t

Reply:
ptrdiff_t is `long int`, so gcc still complains about signedness

Reply:
Then `size_t`.